.github/workflows/release.yaml

```diff
@@ -1,10 +1,14 @@
 name: Release
 on:
-  workflow_dispatch:  # allows triggering a github action manually - see 'Actions' tab
+  push:
+    branches:
+      - main
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    # Important: We use the commit message as a trigger for each job
```

Comment:
Can we put this comment at the top of the file, just once?

There's no way to introduce another sub-target to handle just the commit message?

Reply:
(Follow-up after short clarification chat with @rec)

- We should separate this logic ('check the commit message') into it's own node in the DAG.
- The logic in this new node will either i) continue for a valid commit message or ii) exit for an invalid commit message